### ISSUE TITLE
Long-term Sustainability

### ISSUE THREAD

**Issue by hutson** [andOTP/andOTP#339]
This isn't really a feature request or bug report.

I'm interested in y'alls thoughts on ensuring the long-term sustainability of the `andOTP` project en-light of the fact that it's only maintained in what may very well be just your _free_ time.

Is there a way to provide a small recurring financial contribution to support regular maintenance of this project?

**Comment by flocke** [comment]
andOTP won't receive any major updates anymore as I am re-writing the entire app from scratch. I've written a little bit about it here: https://github.com/andOTP/andOTP/wiki/The-Rewrite

After the initial release of the new app I will try to set something up for reoccuring donations. And you are right, I am only maintaining it in my free time and basically all by my self. Which is why updates are currently slow.

**Comment by RichyHBM** [comment]
Maybe if Github rewards ever takes off you can just use that, but AFAIK this will be a passion project rather than a full time thing, there are only so many things we can add to an OTP app :P 

And dont forget, there is always the donation section in About if you are feeling really generous ;)

**Comment by hutson** [comment]
> andOTP won't receive any major updates anymore as I am re-writing the entire app from scratch. I've written a little bit about it here: https://github.com/andOTP/andOTP/wiki/The-Rewrite

Thank you for bringing the re-write to my attention. I wasn't aware you were working on a substantial re-write of the application, but I'm thrilled to know you're still invested in delivering a great OTP experience to users.

> Maybe if Github rewards ever takes off you can just use that

That's definitely a possibility! 

> AFAIK this will be a passion project rather than a full time thing, there are only so many things we can add to an OTP app :P

:sweat_smile: my concern is using an app and finding that it's no longer supported because the author(s) aren't incentivized to continue (new Android OS, new phones, new protocols, etc.). While money is a tricky thing as an incentive for a passion project, I don't know of any other way to contribute, given that ultimately, I lack interest and motivation to contribute to an OTP app, but I still need the functionality to support my other work.

> And dont forget, there is always the donation section in About if you are feeling really generous ;)

:open_mouth: I hadn't noticed the donation section within the app's About page. I donated a little via PayPal (Though I had to pull the link from [source](https://github.com/andOTP/andOTP/blob/8da709edcfa4b46894e35284e9c96d78d28d3ddd/app/src/main/java/org/shadowice/flocke/andotp/Fragments/AboutFragment.java#L34) since the app just navigates to GitHub).
